src/vivarium_gates_iv_iron/components/observers.py

```diff
@@ -665,7 +665,11 @@ def on_collect_metrics(self, event: Event):
         counts_this_step = {}
         pop = self.population_view.get(event.index)
         pregnancy_change_this_step_pop = pop[
-            pop["pregnancy_state_change_date"] == event.time
+            (pop["pregnancy_state_change_date"] == event.time)
+            & (
+                (pop["pregnancy_status"] == models.MATERNAL_DISORDER_STATE)
+                | (pop["pregnancy_status"] == models.NO_MATERNAL_DISORDER_STATE)
```

Comment:
you can use the keyword `in` here rather than or-ing like this

Reply:
I thought so too, but an error gets raised on that (assuming I'm doing this right)

```  File "/Users/mkappel/git/ssh/vivarium_gates_iv_iron/src/vivarium_gates_iv_iron/components/observers.py", line 669, in on_collect_metrics
    & (pop["pregnancy_status"] in [models.MATERNAL_DISORDER_STATE, models.NO_MATERNAL_DISORDER_STATE])
  File "/Users/mkappel/opt/miniconda3/envs/ivfe_dev/lib/python3.8/site-packages/pandas/core/generic.py", line 1537, in __nonzero__
    raise ValueError(
ValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
```

Reply:
Sorry you can use `in` when using the `.query()` syntax. You use `.isin()` when using this syntax.